### PR TITLE
Perfdash: config for build ranges

### PR DESCRIPTION
**Pull request by krzysied**

### CODE REVIEW

File: perfdash/Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 all: push
 
 # See pod.yaml for the version currently running-- bump this ahead before rebuilding!
-TAG = 1.0
+TAG = 1.01
```
Comment (shyamjvs) [inline perfdash/Makefile]: Why 1.01 instead of 1.1?

**Comment by krzysied** [comment]
/cc mborsz
/cc shyamjvs

I definitely need suggestions for customized build ranges. Default is 100.
Btw, builds.json can also be changed during runtime.

File: perfdash/config/builds.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+	"gce-100Nodes": 100
```
Comment (shyamjvs) [inline perfdash/config/builds.json]: Like I mentioned earlier - let's not try to hardcode this config into perfdash (as then we'll need to rebuild and push everytime we want to change this).

Instead, let's pick this config from https://github.com/kubernetes/test-infra/blob/master/prow/config.yaml (from where we're already picking perfDashTag). You can add a new label there like `perfDashRange` or sth - and we can easily change this range then by just a single PR changing that value.

**Comment by shyamjvs** [comment]
I've left above a high-level comment about this approach.

File: perfdash/Makefile (hunk 1 above)
Comment (krzysied) [reply perfdash/Makefile]: For some reason I thought that we are always using two digits for minor version. I will change it to 1.1 if I assumed incorrectly.

File: perfdash/config/builds.json (hunk 2 above)
Comment (krzysied) [reply perfdash/config/builds.json]: Good idea. I will change my code.

File: perfdash/Makefile (hunk 1 above)
Comment (shyamjvs) [reply perfdash/Makefile]: It's usually just a serially increasing number for the minor version. (btw - naming it as 1.01 wouldn't let us have more than 99 minor versions in any decent way :)

File: perfdash/config/builds.json (hunk 2 above)
Comment (krzysied) [reply perfdash/config/builds.json]: Done. `perfDashBuilds` is a new label for specifying builds.

File: perfdash/config.go [hunk 3]
```diff
@@ -42,6 +43,7 @@ type TestDescription struct {
 type Tests struct {
 	Prefix       string
 	Descriptions map[string]TestDescription
+	Builds       int
```
Comment (shyamjvs) [inline perfdash/config.go]: Maybe `NumBuilds` or `BuildsCount`?

File: perfdash/config.go [hunk 4]
```diff
@@ -188,6 +190,19 @@ func getProwConfig() (Jobs, error) {
 				thisPeriodicConfig.Descriptions = jobTypeToDescriptions[jobType]
 				continue
 			}
+			if strings.HasPrefix(tag, "perfDashBuilds:") {
+				split := strings.SplitN(tag, ":", 2)
+				i, err := strconv.Atoi(strings.TrimSpace(split[1]))
+				if err != nil {
+					fmt.Fprintf(os.Stderr, "warning: unparsable builds - %v\n", split[1])
+				}
+				if i < 1 {
```
Comment (shyamjvs) [inline perfdash/config.go]: Shouldn't this be 0 (if you're excluding negative values)?

Comment (shyamjvs) [reply perfdash/config.go]: Or change the error msg below.

File: perfdash/config.go (hunk 3 above)
Comment (krzysied) [reply perfdash/config.go]: Ok. Changing to `BuildsCount`.

File: perfdash/config.go (hunk 4 above)
Comment (krzysied) [reply perfdash/config.go]: You are right. Should be non-negative.

File: perfdash/config.go [hunk 5]
```diff
@@ -188,6 +190,19 @@ func getProwConfig() (Jobs, error) {
 				thisPeriodicConfig.Descriptions = jobTypeToDescriptions[jobType]
 				continue
 			}
+			if strings.HasPrefix(tag, "perfDashBuilds:") {
```
Comment (shyamjvs) [inline perfdash/config.go]: Also here perfDashBuildsCount

Comment (krzysied) [reply perfdash/config.go]: done

**Pull request merged**